source/agora/node/Ledger.d

```diff
@@ -193,10 +194,10 @@ public class Ledger
                 Block block = this.storage.readBlock(Height(height));
 
                 // Make sure our data on disk is valid
-                if (auto fail_reason = this.validateBlock(block))
+                if (auto fail_reason = this.validateBlock(block, file, line))
                     throw new Exception(
-                        "A block loaded from disk is invalid: " ~
-                        fail_reason);
+                        format!"[%s:%s]: A block loaded from disk is invalid: %s"
+                        (file, line, fail_reason));
```

Comment:
Why ? We get a stack trace anyway which includes this information.

Reply:
I don't think it was included in the case when I came across a unit test failing with an assert here. I should have copied the stack trace for reference. I will need to try and reproduce it.

Reply:
Make sure to use ldc from `~/dlang` not Homebrew's

Reply:
ok will revert.